src/urllib3/connectionpool.py

```diff
@@ -461,7 +474,9 @@ def _make_request(
 
         # Receive the response from the server
         try:
-            httplib_response = conn.getresponse()
+            if response_kw is None:
+                response_kw = {}
+            response = conn.getresponse(retries, **response_kw)
```

Comment:
Instead of using `response_kw` we should explicitly define which parameters are being forwarded to `getresponse()` so we can make them a part of the `BaseHTTPConnection` API in the future. You can start with:

- retries
- connection
- pool
- request_method
- request_url

If there are extras that fail we can add them too.

Reply:
This is fixed